metadata-ingestion/src/datahub/ingestion/source/sagemaker_processors/models.py

```diff
@@ -333,6 +384,24 @@ def get_model_wu(
             model_metrics,
         ) = self.match_model_jobs(model_details)
 
+        # resolve groups that the model is a part of
+        model_uri_groups = self.lineage.model_uri_to_groups.get(model_uri, set())
+        model_image_groups = self.lineage.model_image_to_groups.get(model_image, set())
+
+        model_group_arns = model_uri_groups | model_image_groups
+
+        model_group_names = sorted(
+            [self.group_arn_to_name[x] for x in model_group_arns]
+        )
+        model_group_urns = [
+            builder.make_ml_model_group_urn("sagemaker", x, self.env)
+            for x in model_group_names
+        ]
+
+        model_browsepaths = [
```

Comment:
Does this cover the case where groups may not exist?

Reply:
Will add!